interface/language/csv/commit_csv.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Kevin Yeh <kevin.y@integralemr.com>
+ * @author  Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2014 Kevin Yeh <kevin.y@integralemr.com>
+ * @copyright Copyright (c) 2021 Rod Roark <rod@sunsetsystems.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+// This script is ajax-loaded into a div by validate_csv.php, so much of
+// the normal framework does not apply.
```

Comment:
does it make sense for scripts that only return json to do this:
```header('Content-Type: application/json');```
(figured this may decrease risk of cross scripting attacks)

Reply:
Hmm, I'll try and see if it works.